include/records.hrl

```diff
@@ -60,3 +60,13 @@
                    orig_query :: iodata(),
                    param_count :: integer(),
                    warning_count :: integer()}).
+
+-ifdef(OTP_RELEASE). %% this implies 21 or higher
+-define(EXCEPTION(Class, Reason, Stacktrace), Class:Reason:Stacktrace).
```

Comment:
Nice macros, but they don't belong in the records.hrl file. It is misleading and they are used in some files where the protocol records are not needed. Please move to a new file `exception.hrl` file or similar.